agtern/gui/frames/internship_detail_frame.py

```diff
@@ -71,4 +71,9 @@ def _on_apply_now_bttn_click(self):
         button is clicked.
         """
         print('"Apply now" was clicked and the link is ' \
-              f'"{self._var_internship_link.get()}"')
\ No newline at end of file
+              f'"{self._var_internship_link.get()}"')
+
+        link = self._var_internship_link.get()
+        if(link): # Ensure not empty 
```

Comment:
```suggestion
        if link:  # Ensure not empty
```
Complying with PEP8 standards: at least 2 spaces for inline comments and no parentheses in `if` statement